lib/net.js

```diff
@@ -899,24 +899,18 @@ function connect(self, address, port, addressType, localAddress, localPort) {
 }
 
 
-Socket.prototype.connect = function(options, cb) {
+Socket.prototype.connect = function() {
+  const args = new Array(arguments.length);
+  for (var i = 0; i < arguments.length; i++)
+    args[i] = arguments[i];
+  // TODO(joyeecheung): use destructuring when V8 is fast enough
```

Comment:
@vhain (hit the button too soon)..but yeah you can push it here(should be semver-patch) or add it in a separate PR(semver-minor).